### DISCUSSION TITLE
Are there configuration file equivalent to CLI arguments `--ext`, `--fix`, `--ignore-path`?

### DISCUSSION THREAD

**Discussion by zhao-li** [eslint/eslint#16209] [Q&A] [answered]
Hi 👋

VueJS auto generates this command to use eslint: `eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore`

I want to push these configurations into a configuration file and then just run a more simple command of: `eslint`

However, looking at these docs: https://eslint.org/docs/latest/user-guide/command-line-interface#--fix
I can't figure out the equivalent setting in my `.eslintrc.cjs`.

Is there an equivalent setting supported by `.eslintrc.cjs` that I can use to replace using these CLI arguments?:
- `--ext`
- `--fix`
- `--ignore-path`

Or is there a documentation that lists the available options of a `.eslintrc.cjs` file?

Thank you for your time 🙏

**Comment by nzakas** [comment, marked as answer]
You can use overrides to mimic —ext.

There are no config file equivalents for the others. 

Please see https://eslint.orgs/docs

**Reply by zhao-li** [reply]
Thanks @nzakas for letting me know that there is not something exactly what I am looking for. 🙏
